### DISCUSSION TITLE
Welcome to Admin Discussions!

### DISCUSSION THREAD

**Discussion by qunabu** [EscolaLMS/Admin#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amittyuj** [comment]
Hi Amit here can any one please provide working Repository for Admin,
Also please provide steps to install and run the admin

**Reply by qunabu** [reply]
You need to have API running in first place to have Admin attached somehow

see 
- https://github.com/EscolaLMS/Create-LMS-App 
- https://docs.wellms.io/
